planner/core/memtable_predicate_extractor.go

```diff
@@ -606,7 +606,17 @@ func (e *ClusterLogTableExtractor) Extract(
 }
 
 func (e *ClusterLogTableExtractor) explainInfo(p *PhysicalMemTable) string {
-	return ""
+	if e.SkipRequest {
+		return "skip_request: true"
+	}
+	startTime, endTime := e.StartTime, e.EndTime
+	st := time.Unix(0, startTime*1e6)
+	et := time.Unix(0, endTime*1e6)
+	return fmt.Sprintf("Start_time:%v, end_time:%v, node_types:[%s], instances:[%s], log_levels:[%s]",
+		st.In(p.ctx.GetSessionVars().StmtCtx.TimeZone).Format(MetricTableTimeFormat),
+		et.In(p.ctx.GetSessionVars().StmtCtx.TimeZone).Format(MetricTableTimeFormat),
+		ExtractStringFromStringSet(e.NodeTypes), ExtractStringFromStringSet(e.Instances), ExtractStringFromStringSet(e.LogLevels),
```

Comment:
Please add `len(e.NodeTypes) > 0` check.

It's hard to see the difference of `level`at below:

```sql
information_schema> desc select * from `CLUSTER_LOG`;
+----------------+----------+------+-----------------------------------------------------------------------------------------------------------------------------+
| id             | estRows  | task | operator info                                                                                                               |
+----------------+----------+------+-----------------------------------------------------------------------------------------------------------------------------+
| MemTableScan_4 | 10000.00 | root | table:CLUSTER_LOG, Start_time:1970-01-01 08:00:00, end_time:1970-01-01 08:00:00, node_types:[], instances:[], log_levels:[] |
+----------------+----------+------+-----------------------------------------------------------------------------------------------------------------------------+
1 row in set
Time: 0.011s
information_schema> desc select * from `CLUSTER_LOG` where level = ''
+----------------+----------+------+---------------------------------------------------------------------------------------------------------------------------------+
| id             | estRows  | task | operator info                                                                                                                   |
+----------------+----------+------+---------------------------------------------------------------------------------------------------------------------------------+
| MemTableScan_5 | 10000.00 | root | table:CLUSTER_LOG, Start_time:1970-01-01 08:00:00, end_time:1970-01-01 07:59:59.999, node_types:[], instances:[], log_levels:[] |
+----------------+----------+------+---------------------------------------------------------------------------------------------------------------------------------+

```

Reply:
updated.